samples/lj_liquid_distribution.py

```diff
@@ -199,6 +203,8 @@
     r, dist = system.analysis.distribution(type_list_a=distr_type_list_a, type_list_b=distr_type_list_b,
                                            r_min=distr_r_min, r_max=distr_r_max, r_bins=distr_r_bins,
                                            log_flag=distr_log_flag, int_flag=distr_int_flag)
+    print(dist)
```

Comment:
Debug stuff

Reply:
Revoked changes there, thanks for spotting...